arangod/Aql/MaterializeExecutor.cpp

```diff
@@ -118,14 +205,31 @@ arangodb::aql::MaterializeExecutor<T>::produceRows(
       }
     }
 
-    // FIXME(gnusi): use rocksdb::DB::MultiGet(...)
-    written =
-        collection->getPhysical()
-            ->read(&_trx,
-                   LocalDocumentId(input.getValue(docRegId).slice().getUInt()),
-                   callback, ReadOwnWrites::no)
-            .ok();
-
+    if constexpr (std::is_same<T, std::string const&>::value) {
```

Comment:
```suggestion
    if constexpr (std::is_same_v<T, std::string const&>) {
```

But better to have flag in base class

Reply:
it is done like that already. Just not pushed )) PR is still WIP

Reply:
Ok)